lib/fastlane/plugin/mango/actions/run_dockerized_task_action.rb

```diff
@@ -202,6 +203,12 @@ def self.available_options
                                        description: 'A bool. True for vnc_enabled False for vnc_disabled',
                                        type: Boolean,
                                        default_value: true,
+                                       optional: true),
+
+          FastlaneCore::ConfigItem.new(key: :emulator_language,
```

Comment:
This will be executed as well even we do not have the `emulator_language` param

Reply:
this is just a config item, it does not get executed but it will allow :emulator_language as a parameter